### ISSUE TITLE
Erroring out while compiling `open_exchange_rates.ex` - even when its turned off in config.exs

### ISSUE THREAD

**Issue by vraravam** [kipcole9/money#2]
I am using the latest version of both `ex_money` and `ex_cldr`. This is the corresponding line from my `mix.lock` file.

```
  "ex_cldr": {:hex, :ex_cldr, "0.0.13", "58843aef84ae5b8ae975335fa8aba46602ef266a3a67f88d069348362cb4b23b", [:mix], [{:decimal, "~> 1.1", [hex: :decimal, optional: false]}, {:poison, "~> 2.1", [hex: :poison, optional: false]}]},
  "ex_money": {:hex, :ex_money, "0.0.10", "f49b1749183da64f6bf92c480cf854ad0d023b843b080e618454ca59fa3d241f", [:mix], [{:ex_cldr, "~> 0.0.13", [hex: :ex_cldr, optional: false]}, {:httpoison, "~> 0.10.0", [hex: :httpoison, optional: false]}]},
```

Even with the following configuration in my `config.exs`:
config :ex_cldr,
  default_locale: "en",
  locales: ["en-IN", "en", "eu"],
  gettext: ContactDemo.Gettext

config :ex_money,
  exchange_rate_service: false
```

I am still getting this error: 
<img width="958" alt="screen shot 2016-12-11 at 9 30 57 am" src="https://cloud.githubusercontent.com/assets/69629/21078025/e44c6cf8-bf85-11e6-9560-55c3879ce6be.png">

**Comment by kipcole9** [comment]
Vijay, yes, its a bug.  If you add a key with a dummy open exchange rates app_id the code will compile.  The next version will not start the service by default and will not require an open exchange rates id to be configured.  I expect to have that completed by Tuesday 13th.

In your config file:
```
config :ex_money,
  exchange_rate_service: false,
  open_exchange_rates_app_id: "dummy"
```
I'll leave the issue open to ensure I get this done quickly.

**Comment by kipcole9** [comment]
Fixed in v0.0.11.
